config.py

```diff
@@ -34,6 +34,8 @@ def set_debug_flag(debug):
 TRACKER_PASSWORD_LENGTH_MIN = config_tracker.getint('password_length_min')
 TRACKER_PASSWORD_LENGTH_MAX = config_tracker.getint('password_length_max')
 TRACKER_SUMMARY_LENGTH_MAX = config_tracker.getint('summary_length_max')
+TRACKER_OTP_FORMAT = config_tracker['otp_format']
+TRACKER_ENABLE_TWO_FACTOR = config_tracker.getboolean('enable_two_factor')
```

Comment:
both should be named `enforce_two_factor` as 2FA should always be possible its just a matter of configuration if it is enforced on everyone or not.